src/net.h

```diff
@@ -636,8 +644,10 @@ class CNode
         try {
             BeginMessage(pszCommand);
             ssSend << a1 << a2;
-            EndMessage();
-        } catch (...) {
+            EndMessage(pszCommand);
+        }
+        catch (...)
+        {
```

Comment:
Please do not mix formatting change with actual code changes.

Looks like this should be done using a ScopeGuard, if you really want to touch it.